### DISCUSSION TITLE
Cloudreve配置onlyoffice真的是累死了，还是配置不成功

### DISCUSSION THREAD

**Discussion by a506488043** [cloudreve/cloudreve#3268] [General]
**Is your feature request related to a problem? Please describe.**
onlyoffice docker
`docker run -i -t -d -p 9980:80 --restart=always \
-v /mnt/256G/docker/onlyoffice/logs:/var/log/onlyoffice \
-v /mnt/256G/docker/onlyoffice/data:/var/www/onlyoffice/Data \
-v /mnt/256G/docker/onlyoffice/lib:/var/lib/onlyoffice \
-v /mnt/256G/docker/onlyoffice/db:/var/lib/postgresql \
--name onlyoffice --restart="always" -e WOPI_ENABLED=true onlyoffice/documentserver`
打开也是正常，就是不能编辑文档
![image](https://github.com/cloudreve/Cloudreve/assets/16434125/dd71c94b-921e-43f5-9cf0-b8e3e6f05bb5)
nextcloud配置onlyoffice 都没这么复杂
![image](https://github.com/cloudreve/Cloudreve/assets/16434125/db0c1a58-fcf0-4582-859d-93e18eaadbb2)

**Comment by Rellopn** [comment]
在 docker 部署 onlyoffice ，cloudreve有部署到本机，你的面板管理-参数设置-站点信息-站点 URL要设置成你本机的局域网ip。

**Comment by yunqingan** [comment]
@Rellopn very good!!!

**Comment by yuanshangyujingshierlou** [comment]
> @Rellopn very good!!!

什么意思呢，大佬能说的详细点吗

**Comment by Phlphat746** [comment]
Ok  love you too 💕😽

ในวันที่ อา. 24 มี.ค. 2024 16:12 น. Nancy Gleason ***@***.***>
เขียนว่า:

**Comment by zzti** [comment]
> 在 docker 部署 onlyoffice ，cloudreve有部署到本机，你的面板管理-参数设置-站点信息-站点 URL要设置成你本机的局域网ip。

这样只能在局域网内使用IP地址访问,在公网使用域名就不行了.....

**Comment by Gosline01** [comment]
> **Is your feature request related to a problem? Please describe.** onlyoffice docker `docker run -i -t -d -p 9980:80 --restart=always \ -v /mnt/256G/docker/onlyoffice/logs:/var/log/onlyoffice \ -v /mnt/256G/docker/onlyoffice/data:/var/www/onlyoffice/Data \ -v /mnt/256G/docker/onlyoffice/lib:/var/lib/onlyoffice \ -v /mnt/256G/docker/onlyoffice/db:/var/lib/postgresql \ --name onlyoffice --restart="always" -e WOPI_ENABLED=true onlyoffice/documentserver` 打开也是正常，就是不能编辑文档 ![image](https://github.com/cloudreve/Cloudreve/assets/16434125/dd71c94b-921e-43f5-9cf0-b8e3e6f05bb5) nextcloud配置onlyoffice 都没这么复杂 ![image](https://github.com/cloudreve/Cloudreve/assets/16434125/db0c1a58-fcf0-4582-859d-93e18eaadbb2)

在windows部署上安装存在这个问题，在ubuntu部署上不存在这个问题。

**Comment by ouyangtiedan** [comment]
我在linux上也遇到了同样的问题

**Comment by 804260247** [comment]
相同的情况，不过现在有AI很方便，把你的情况直接丢给AI就能出来解决方案，我这边已经解决了，不过我的情况更复杂（cloudreve、onlyoffice、nginx分别在3台NAS上...）
